inventory.yml.sample

```diff
@@ -122,8 +122,13 @@ all:
       }}"
     mirror_certificate: "{{ lookup('file', local_mirror_certificate_path) }}"
 
-    openshift_version: "{{ openshift_full_version.split('.')[:2] | join('.') }}"
+    openshift_full_version_mapping:
+      '4.6': 4.6.16
+      '4.7': 4.7.33
+      '4.8': 4.8.14
+      '4.9': 4.9.0
 
+    openshift_full_version: "{{ openshift_full_version_mapping[openshift_version | string] }}"
```

Comment:
How will this interact with an older inventory? If someone uses an old inventory, the new `openshift_full_version_mapping` which is used in `get_image_hash` will not exist. Add a helpful error message for this case, to ease the transition for people with an existing inventory they need to update.